### PR TITLE
Feature/rchain calculation persist results

### PR DESCRIPTION
**Pull request by SteRiccio**

### CODE REVIEW

File: server/modules/analysis/service/_rChain/_files/system/rFileResetResults.js [hunk 1]
```diff
@@ -11,9 +17,23 @@ export default class RFileResetResults extends RFileSystem {
...
+    const viewNamesPrexif = ResultView.getViewNamesPrefix(chainUuid)
+    const views = await DbRepository.fetchMaterializedViewsBySchema(SchemaRdb.getName(this.rChain.surveyId))
+
+    for (const view of views) {
+      const viewName = MaterializedView.getViewName(view)
+      if (viewName.startsWith(viewNamesPrexif)) {
+        const dropView = dbSendQuery(`DROP MATERIALIZED VIEW IF EXISTS \\"${viewName}\\"`)
+        await this.appendContent(dropView)
+      }
+    }
```
Comment (minotogna) [inline server/modules/analysis/service/_rChain/_files/system/rFileResetResults.js]: do we still need this?

File: server/db/repository/dbRepository.js [hunk 2]
```diff
@@ -0,0 +1,23 @@
+import camelize from 'camelize'
```
Comment (minotogna) [inline server/db/repository/dbRepository.js]: do we still need this?

File: server/db/model/materializedView.js [hunk 3]
```diff
@@ -0,0 +1,7 @@
+import * as R from 'ramda'
```
Comment (minotogna) [inline server/db/model/materializedView.js]: do we still need this?

File: core/i18n/resources/en.js [hunk 4]
```diff
@@ -694,7 +695,10 @@ $t(common.cantUndoWarning)`,
     SurveyPropsPublishJob: 'Survey Props Publish',
     SurveyPublishJob: 'Survey Publish',
     SurveyPublishPerformJob: 'Survey Publish Perform',
-    SurveyRdbGeneratorJob: 'Survey RDB Generator',
+    SurveyRdbCreationJob: 'Survey RDB Creation',
+    SurveyRdbDataTablesAndViewsCreationJob: 'Survey RDB Data Tables And Views Creation',
```
Comment (minotogna) [inline core/i18n/resources/en.js]: do we have only 1 job?

File: server/modules/collectImport/service/collectImport/collectImportJob.js [hunk 5]
```diff
@@ -25,7 +26,8 @@ export default class CollectImportJob extends Job {
       new NodeDefsImportJob(),
       new SurveyDependencyGraphsGenerationJob(),
       new RecordsImportJob(),
-      new SurveyRdbGeneratorJob(),
+      new SurveyRdbCreationJob(),
+      new AnalysisUserCreationJob(),
```
Comment (minotogna) [inline server/modules/collectImport/service/collectImport/collectImportJob.js]: can this be a sub job of SurveyRdbCreationJob?

File: core/i18n/resources/en.js (hunk 4 above)
Comment (SteRiccio) [reply core/i18n/resources/en.js]: ok

**Pull request merged**